pandas/tests/reshape/concat/test_concat.py

```diff
@@ -445,6 +446,54 @@ def test_concat_ordered_dict(self):
         tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize("index_maker", tm.index_subclass_makers_generator())
+@pytest.mark.parametrize("join", ["inner", "outer"])
+def test_concat_duplicates_error(index_maker, join):
+    # if index_maker is tm.makeMultiIndex:
+    # TODO: This generator only makes Indexs of size 4
```

Comment:
what is this about?

Reply:
https://github.com/pandas-dev/pandas/blob/0976c4ca0276a333346619c5a9e3c92631ec7006/pandas/_testing/__init__.py#L491-L492

This function only makes a MultiIndex with length 4, ignoring the `k` parameter

Reply:
#38795